src/hyperfine/app.rs

```diff
@@ -129,6 +129,21 @@ fn build_app() -> App<'static, 'static> {
                      This performs benchmarks for 'sleep 0.3', 'sleep 0.5' and 'sleep 0.7'.",
                 ),
         )
+        .arg(
+            Arg::with_name("parameter-list")
+                .long("parameter-list")
+                .short("L")
+                .takes_value(true)
+                .allow_hyphen_values(true)
+                .value_names(&["VAR", "VALUES"])
+                .conflicts_with("parameter-scan")
+                .help(
+                    "Perform benchmark runs for each value in the comma-seperated list VALUES. \
+                     Replaces the string '{VAR}' in each command by the current parameter value\
+                     .\n\nExample:  hyperfine -L threads 1,2,4 'make -j {threads}'\n\n\
```

Comment:
Can we find a simple example with non-numeric parameter values?

Reply:
Perhaps `hyperfine -L file a,b,c "gcc {file}.cpp"` ?

Reply:
how about:
```c++
hyperfine -L compiler gcc,clang "{compiler} -O3 -Wall main.c"
```

Reply:
That's actually really useful when I compare different versions of my programs, because I will not have to repeat the arguments over and over again. For example:
``` bash
hyperfine -L version 6.0,7.1,7.2,7.4,master "./fd-{version} -HI 'my-pattern' /some/directory"
```

Reply:
I like the different program versions a lot